### PR TITLE
ts/pie-series

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

File: ts/Series/Pie/PiePoint.ts [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+ * */
+
+interface PiePoint {
+    connectorShapes: Record<string, PiePoint.ConnectorShapeFunction>;
+    labelDistance: number;
+    name: string;
+    options: PiePointOptions;
+    series: PieSeries;
+}
+extend(PiePoint.prototype, {
```
Comment (TorsteinHonsi) [inline ts/Series/Pie/PiePoint.ts:301]: Why do we do extend in this strange manner instead of just putting it in the class?

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]

File: ts/Series/Pie/PiePoint.ts (hunk 1 above)
Comment (bre1470) [reply ts/Series/Pie/PiePoint.ts:301]: In the class they need a default value in the constructor. A later `init` call by chart is not sufficient.

Comment (bre1470) [reply ts/Series/Pie/PiePoint.ts:301]: Also to keep compatibility, the properties in the `extend` call continue to be set on the prototype. In a class they would be set for each instance separately, which can break code, fiddles, tests, and wraps.

Comment (bre1470) [reply ts/Series/Pie/PiePoint.ts:301]: I will improve the situation with `any` casts. It has to be refactored anyway later on.

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**